homeassistant/components/tibber/manifest.json

```diff
@@ -3,7 +3,7 @@
   "domain": "tibber",
   "name": "Tibber",
   "documentation": "https://www.home-assistant.io/integrations/tibber",
-  "requirements": ["pyTibber==0.25.6"],
+  "requirements": ["pyTibber==0.26.0"],
```

Comment:
Just use `asyncio.create_task` instead here:
https://github.com/Danielhiversen/pyTibber/blob/001e9d059bf22c7d0c0d2b1b436e2538a9f7d9da/tibber/tibber_home.py#L447

And use `asyncio.create_task` instead of `asyncio.ensure_future`. The latter is low level.

https://docs.python.org/3/library/asyncio-future.html#asyncio.ensure_future